db/migrate/20151229184253_create_token_auth_synchronizable_resources.rb

```diff
@@ -13,3 +13,13 @@ def change
     end
   end
 end
+
+if ActiveRecord::Migration.respond_to? :[]
+  class CreateTokenAuthSynchronizableResources < ActiveRecord::Migration[4.2]
```

Comment:
In this case, what is the reasoning behind the conditional block?  Seems like everything is the same.